File: SEMESTRE_6/TEMAS_ESPECIALES_COMPUTACION/aplicacion_1/app/src/main/java/com/example/aplicacion_1/SaludoActivity.java
package com.example.aplicacion_1;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;


public class SaludoActivity extends AppCompatActivity{
    @SuppressLint("SetTextI18n")

    private TextView saludo;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_saludo);

        //Localizar los controles
        TextView saludo = (TextView) findViewById(R.id.txtSaludo);

        //Recuperamos la información pasada en el intent
        Bundle bundle = this.getIntent().getExtras();

        //Construimos el mensaje a mostrar
        assert bundle != null;
        saludo.setText("Hola " + bundle.getString("NOMBRE") + bundle.getString("AP_PATERNO") + bundle.getString("AP_MATERNO") );
    }
}

File: SEMESTRE_6/TEMAS_ESPECIALES_COMPUTACION/aplicacion_2/app/src/main/java/com/example/aplicacion_2/SaludoActivity.kt
package com.example.aplicacion_2

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class SaludoActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        lateinit var txtSaludo : TextView

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_saludo)
        //Obtenemos una referencia a los controles de la interfaz
        txtSaludo = findViewById(R.id.txtSaludo)
        //Recuperamos la información pasada en el intent
        val saludo = intent.getStringExtra("NOMBRE")
        //Construimos el mensaje a mostrar
        txtSaludo.text = "Hola $saludo"
    }

}